handler/webhook_test.go

```diff
@@ -0,0 +1,68 @@
+package handler_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/forjadev/gun-organization/handler"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWebhookReceiverHandle_Success(t *testing.T) {
```

Comment:
We should change all the places that mention the old name of the function to the new name, not only the one mentioned in this review thread (to avoid polluting and repeating too much with the same message during the review).